component/authentication-endpoint/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update these license headers? Check all the new files.

Reply:
Fixed with 208f4a5beabb544f608585d16512c1b9e45a2117